### PR TITLE
Bugfix for Helm.

### PR DESCRIPTION
**Pull request by yhontyk**
A small bugfix for Helm docs:

- branch 4.3 and 4.4
- reviewed by the QE/SMEs
- cc @bergerhoffer

### CODE REVIEW

File: modules/helm-creating-a-custom-helm-chart-on-openshift.adoc [hunk 1]
```diff
@@ -28,10 +28,16 @@ $ cd redhat-helm-charts/alpha/nodejs-ex-k/
 +
 [source,yaml]
 ----
-apiVersion: v2
-name: Node.js
-description: A Helm chart for OpenShift
+apiVersion: v2 <1>
+name: nodejs-ex-k <2>
+description: A Helm chart for Openshift <3>
```
Comment (bergerhoffer) [inline modules/helm-creating-a-custom-helm-chart-on-openshift.adoc]: This changed "OpenShift" to "Openshift". It should stay "OpenShift".

**Review by bergerhoffer — COMMENTED** [review]
Just one minor typo fix.

File: modules/helm-creating-a-custom-helm-chart-on-openshift.adoc (hunk 1 above)
Comment (boczkowska) [reply modules/helm-creating-a-custom-helm-chart-on-openshift.adoc]: Thank you for the review! 
Fixed.

**Comment by bergerhoffer** [comment]
Thanks! Merging.

**Pull request merged**